cid/BoardConfig.mk

```diff
@@ -91,6 +91,6 @@ WIFI_BYPASS_FWRELOAD      := true
 BOARD_BLUETOOTH_BDROID_BUILDCFG_INCLUDE_DIR := device/boundary/cid/
 
 include device/boundary/sepolicy.mk
-include hardware/sierra/sepolicy.mk
+#include hardware/sierra/sepolicy.mk
```

Comment:
Same here, don't comment, just remove. What happened to the sierra sepolicy, there's none of them now? I doubt that's the case.
Also, this commit needs to be squashed with the previous one.
Finally, please explain the changes in the commit log.